servers/rendering/renderer_rd/framebuffer_cache_rd.h

```diff
@@ -253,8 +253,7 @@ class FramebufferCacheRD : public Object {
 	}
 
 	RID get_cache_multipass(const Vector<RID> &p_textures, const Vector<RD::FramebufferPass> &p_passes, uint32_t p_views = 1) {
-		uint32_t h = hash_murmur3_one_32(p_views);
-		h = hash_murmur3_one_32(p_textures.size());
+		uint32_t h = hash_murmur3_one_32(p_textures.size());
```

Comment:
Probably should be:
```suggestion
		uint32_t h = hash_murmur3_one_32(p_views);
		h = hash_murmur3_one_32(p_textures.size(), h);
```

Reply:
It looks like this should be investigated more thoroughly. Will remove it from this PR.